hab_gui/widgets/alias_button.py

```diff
@@ -0,0 +1,57 @@
+import logging
+import subprocess  # noqa: F401
+
+from Qt import QtCore, QtGui, QtWidgets
+
+logger = logging.getLogger(__name__)
+
+
+class AliasButton(QtWidgets.QToolButton):
+    """Create a QToolButton which will launch a specified alias via a subprocess.
+
+    Args:
+        cfg (hab.Resolver.resolve): The config dict which contains URI related data.
```

Comment:
`config dict` is incorrect, it should be `config object` or something. `hab.Resolver.resolve` is not a data type its a function that returns a `hab.parsers.config.Config` or `hab.parsers.flat_config.FlatConfig` class.

Reply:
Yeah I meant to ask about this.  Thanks for clarifying.